packages/server-wallet/docs/index.md

```diff
@@ -0,0 +1,183 @@
+# What is a state channel wallet?
+
+A node in a state channel network. See [nitro protocol](https://docs.statechannels.org/`).
+
+More concretely, a state channel wallet _is_:
+
+- a queryable store of keys, states and signatures
+- a run-loop, triggered by
+
+  - blockchain events
+  - API calls
+  - messages from counterparties
+
+  that
+
+  - signs states and transactions
+  - stores them
+  - passes them to external actors
+
+- an event emitter
+
+We usually talk of “The App” as the actor that makes the API calls and binds to the events. The wallet can be a library or run in a separate process to The App (e.g. on a server, or run in an iframe).
+
+A state channel wallet _is not_:
+
+- equipped with a graphical user interface (although this might be a desired plugin)
+
+A state channel wallet has the following _runtime dependencies_:
+
+- a message delivery service (deferred to The App)
```

Comment:
I think this wording is compatible with the status quo API and the "core CM" proposed API.

```suggestion
- a message delivery service (provided by The App)
```